crypto/trace.c

```diff
@@ -124,15 +124,14 @@ static const struct trace_category_st trace_categories[] = {
     TRACE_CATEGORY_(INIT),
     TRACE_CATEGORY_(TLS),
     TRACE_CATEGORY_(TLS_CIPHER),
-    TRACE_CATEGORY_(ENGINE_CONF),
+    TRACE_CATEGORY_(CONF),
     TRACE_CATEGORY_(ENGINE_TABLE),
     TRACE_CATEGORY_(ENGINE_REF_COUNT),
     TRACE_CATEGORY_(PKCS5V2),
     TRACE_CATEGORY_(PKCS12_KEYGEN),
     TRACE_CATEGORY_(PKCS12_DECRYPT),
     TRACE_CATEGORY_(X509V3_POLICY),
     TRACE_CATEGORY_(BN_CTX),
-    TRACE_CATEGORY_(CONF),
```

Comment:
Failing to add an entry here will be another frequent source of error, because it is detected only at runtime. We could add a compile time assert for the `countof(trace_categories)`, what do you think?

Reply:
That is a good idea.  Are you willing to make that PR?

Reply:
I can do it, but not before this evening.